test/support/utils_test_base.h

```diff
@@ -393,7 +393,7 @@ void update_data(TTestDataTransfer& helper, Args&& ...args)
 #endif // TEST_DPCPP_BACKEND_PRESENT
 
 #if TEST_DPCPP_BACKEND_PRESENT
-#define DEFINE_TEST_CONSTRUCTOR(TestClassName)                                                                    \
+#define DEFINE_TEST_CONSTRUCTOR(TestClassName, ScaleStepValue, ScaleMaxNValue)                                    \
```

Comment:
```suggestion
#define DEFINE_TEST_PERM_IT_CONSTRUCTOR(TestClassName, ScaleStepValue, ScaleMaxNValue)                                 \
```

Reply:
I dont understand this comment.  Permutation iterator has its own definition in `test/parallel_api/iterator/permutation_iterator_common.h`

Reply:
You have about 7 extra spaces before '\' char in this line. Line format is broken.